drizzlepac/haputils/svm_quality_graphics.py

```diff
@@ -444,13 +523,193 @@ def build_crossmatch_plots(xmatchDF, data_cols,
+                             'y_3x3_sigma-clipped_mean',
+                             tooltips_list,
+                             hover_columns)
+    plots.append(plot.fig)
+
+    # 8: 3x3 sigma-clipped median: plot x vs y values for all filters
+    plot = make_scatter_plot(xm_cds,
+                             'Comparision - reference 3 x 3' + u'\u03c3' + '-clipped median separation',
+                             '3 x 3' + u'\u03c3' + '-clipped median X separation (pixels)',
+                             '3 x 3' + u'\u03c3' + '-clipped median Y separation (pixels)',
+                             'x_3x3_sigma-clipped_median',
+                             'y_3x3_sigma-clipped_median',
+                             tooltips_list,
+                             hover_columns)
+    plots.append(plot.fig)
+
+    # 9: 3x3 sigma-clipped standard deviation: plot x vs y values for all filters
+    plot = make_scatter_plot(xm_cds,
+                             'Comparision - reference separation 3 x 3' + u'\u03c3' + '-clipped standard deviation',
+                             '3 x 3' + u'\u03c3' + '-clipped X SD (pixels)',
+                             '3 x 3' + u'\u03c3' + '-clipped Y SD (pixels)',
+                             'x_3x3_sigma-clipped_rms',
+                             'y_3x3_sigma-clipped_rms',
+                             tooltips_list,
+                             hover_columns)
+    plots.append(plot.fig)
+
+    # 10: delta_X vs delta_Y vector plot for each cross-matched filter and/or x vs delta_x, x vs. delta_y, y
+    # vs. delta_x, y vs. delta_y quad plot each cross-matched filter
+    for line_index in xm_df.index:
+        xm_df2 = xm_df[xm_df['gen_info.dataframe_index'] == line_index]
+        div = Div(text="""
+        <h2>Position vs. Crossmatched interfilter comparison - reference residual values</h2>
+        <p>Comparison image: {}<br/>Reference image: {}<br/>
+        Reference image platescale: {} arcseconds per pixel</p>""".format(xm_df2['comp_imgname'][line_index],
+                                                                          xm_df2['ref_imgname'][line_index],
+                                                                          xm_df.ref_image_platescale[0]))
+        plots.append(div)
+        xm_cds = ColumnDataSource(xm_df2)
+        cds2_dict = {"x": xm_cds.data["ref_catalog.xcentroid_ref"][0],
+                     "y": xm_cds.data["ref_catalog.ycentroid_ref"][0],
+                     "dx": xm_cds.data["comp-ref_x_seperation"][0],
+                     "dy": xm_cds.data["comp-ref_y_seperation"][0]}
+        residsCDS = ColumnDataSource(cds2_dict)
+        quad_plot = make_quad_scatter_plot(residsCDS)
+        plots += quad_plot
+
+        plot = build_vector_plot(residsCDS, title='Vector plot of Comparison - Reference residuals',
+                                 x_label='X (pixels)', y_label='Y (pixels)', color='blue')
+        plots.append(plot.fig)
```

Comment:
Doing this will result in a single HTML file which may contain thousands of plots, all with their data included in the HTML file.  The pipeline astrometry plots result in the a large number of vector plots as well, however, they are each saved to their own HTML output file as shown on ssb.stsci.edu/hack/astrometry/testing.  The size of each individual HTML file can also be seen there as well, with the sum of all those HTML files being an approximation of the size of this single HTML file if left as-is.  Take a look at pipeline_graphics. to see how this is handled there and break up this potentially massive output file.

Reply:
Resolved in commit [e32955a](https://github.com/spacetelescope/drizzlepac/pull/741/commits/e32955ad6ed946b806c5c9d7225382127c181e9f)